lib/redis/pipeline.rb

```diff
@@ -122,7 +122,7 @@ def _set(object)
     end
 
     def _command
-      @command
+      @command.map(&:to_s)
```

Comment:
I though so too but no. See how it was still failing: https://travis-ci.org/redis/redis-rb/jobs/50558668

```
test_pipeline_race_condition(TestPipeliningCommands):
IOError: closed stream
    /home/travis/build/redis/redis-rb/lib/redis/connection/ruby.rb:269:in `write'
    /home/travis/build/redis/redis-rb/lib/redis/connection/ruby.rb:269:in `write'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:257:in `write'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:236:in `io'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:255:in `write'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:214:in `process'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:208:in `each'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:208:in `process'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:353:in `ensure_connected'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:207:in `process'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:310:in `logging'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:206:in `process'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:177:in `call_pipelined'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:149:in `call_pipeline'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:279:in `with_reconnect'
    /home/travis/build/redis/redis-rb/lib/redis/client.rb:147:in `call_pipeline'
    /home/travis/build/redis/redis-rb/lib/redis.rb:2140:in `pipelined'
    /home/travis/build/redis/redis-rb/lib/redis.rb:37:in `synchronize'
    /home/travis/.rvm/rubies/ruby-1.8.7-p374/lib/ruby/1.8/monitor.rb:242:in `mon_synchronize'
    /home/travis/build/redis/redis-rb/lib/redis.rb:37:in `synchronize'
    /home/travis/build/redis/redis-rb/lib/redis.rb:2132:in `pipelined'
    /home/travis/build/redis/redis-rb/test/pipelining_commands_test.rb:255:in `test_pipeline_race_condition'
    /home/travis/build/redis/redis-rb/test/helper.rb:70:in `run'
```

So restoring the initial client does solve the issue with the command being enqueued in the pipeline, but it was still writing the command in the connection at the wrong time, messing with the currently executed pipeline.